tempesta_fw/t/unit/test_cfg.c

```diff
@@ -45,6 +43,11 @@ TfwCfgMod test_dummy_mod = { .name = "test_dummy_mod" };
 static int
 do_parse_cfg(const char *cfg_text, TfwCfgSpec specs[])
 {
+	int (*tfw_cfg_start_mods)(const char *cfg_text,
+				  struct list_head *mod_list);
+
+	tfw_cfg_start_mods = get_sym_ptr("tfw_cfg_start_mods");
+	BUG_ON(tfw_cfg_start_mods == NULL);
```

Comment:
The line leads to crash on ```make test```:

         [ 2295.161390] tfw_test: TEST_RUN(tfw_addr_pton, recognizes_v4_and_v6_addrs)
        [ 2295.162454] tfw_test: TEST_SUITE_RUN(cfg)
        [ 2295.163099] tfw_test: TEST_RUN(cfg_paresr, invokes_specified_handler)
        [ 2295.168827] ------------[ cut here ]------------
        [ 2295.169004] kernel BUG at /root/tempesta/tempesta_fw/t/unit/test_cfg.c:50!